src/pages/Checkout.tsx

```diff
@@ -300,242 +316,252 @@ export default function Checkout() {
+                <Typography variant="body2" color="text.primary">
+                  Processes
+                </Typography>
+                <Typography variant="body2" color="text.primary" sx={{ fontWeight: 500 }}>
+                  Cost Summary
+                </Typography>
+              </Box>
+
+              <List>
+                {Object.entries(groupServicesByLabel(workflow)).map(([label, service]) => (
+                  <ListItem key={label}>
+                    <ListItemIcon>
+                      <CircleIcon sx={{ fontSize: 8 }} />
+                    </ListItemIcon>
+                    <ListItemText
+                      primary={
+                        <Box
+                          sx={{
+                            display: 'flex',
+                            justifyContent: 'space-between',
+                            alignItems: 'center',
+                            width: '100%'
+                          }}
+                        >
+                          <Box sx={{ display: 'flex', alignItems: 'center', gap: 1 }}>
+                            <span>{label}</span>
+                            {service.count > 1 && (
+                              <Typography
+                                variant="body2"
+                                color="text.secondary"
+                                sx={{ ml: 1, fontSize: '0.875rem' }}
+                              >
+                                (×{service.count})
+                              </Typography>
+                            )}
+                          </Box>
+                          <Typography variant="body2" color="text.secondary">
+                            {formatPriceLabel(service.cost)}
+                          </Typography>
+                        </Box>
+                      }
+                      secondary={
+                        <Box sx={{ mt: 1 }}>
+                          {service.nodes.map((node, index) => (
+                            <Box
+                              key={node.id}
+                              sx={{ mb: index !== service.nodes.length - 1 ? 2 : 0 }}
+                            >
+                              {service.count > 1 && (
+                                <Typography
+                                  variant="body2"
+                                  color="text.secondary"
+                                  sx={{ ml: 2 }}
+                                >
+                                  Service {index + 1}:
+                                </Typography>
+                              )}
+                              {node.data?.formData?.map((param) => (
+                                <Typography
+                                  key={param.id}
+                                  variant="body2"
+                                  color="text.secondary"
+                                  sx={{
+                                    ml: service.count > 1 ? 3 : 2,
```

Comment:
Wow I wasted so much time reverse-engineering/psychoanalyzing this before realizing you didn't even write it :skull: A note from me to the void: this would be better written as nested elements with relative positioning...